tests/python/profiling/simple_forward.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+from mxnet.gluon import nn
+
+
+def simple_forward():
```

Comment:
Is there any plan to add some documentation on how to use this feature as an user?

Reply:
So there's no change required from using the profiler as it exists now, you'll just get more data when using nvprof.

That being said I do want to create documentation specifically describing how to profile with nvprof and the profiler, and describing what kind of output to expect.  I'm ok with deferring this PR until that one is available if you'd prefer.

Reply:
@eric-haibin-lin Added some more information into the existing profiler tutorials.

Reply:
Looks good!